tests/http-pouchdb/test.js

```diff
@@ -0,0 +1,77 @@
+const {PouchDB} = require('pouchdb-plugin-helper/testutils');
+const buildHTTPPouchDB = require('../../packages/node_modules/http-pouchdb');
+const should = require('chai').should();
+
+var HTTPPouchDB = buildHTTPPouchDB(PouchDB, 'http://localhost:5984/');
+var XMLHttpRequest = require('xhr2');
+var Promise = require('bluebird');
+
+describe('isHTTPPouchDB', function () {
+  it('should be set on the HTTPPouchDB object', function () {
+    HTTPPouchDB.isHTTPPouchDB.should.be.ok;
+  });
+  it('should not be set on the PouchDB object', function () {
+    should.not.exist(PouchDB.isHTTPPouchDB);
+  });
+});
+
+describe('constructor', function () {
+  it('should create remote databases for normal db names', function () {
+    var users = new HTTPPouchDB('_users');
+
+    return users.info().then(function (info) {
+      info.should.have.property('db_name');
+    });
```

Comment:
the `constructor` tests are different from the original: https://github.com/pouchdb/http-pouchdb/blob/9b28231f2c40795c045956a6a4b0f3f974c661c9/test/test.js#L25-L39

- For the unit tests, we start our own `pouchdb-server` so the `data_size` does not exist, hence I just check for `db_name`
-`new HTTPPouchDB('http://localhost:5984/_replicator')` is valid with the latest PouchDB, it just encodes all the slashes etc. If that is an error, it’s outside of the scope of `http-pouchdb`